scripts/hail_batch/gnomad_90k_variants_liftover/gnomad_loadings_90k_liftover.py

```diff
@@ -0,0 +1,42 @@
+"""Perform liftover for the gnomAD 90k variants"""
+
+import click
+import hail as hl
+
+GNOMAD_V2_LOADINGS = (
+    'gs://gcp-public-data--gnomad/release/2.1/pca/gnomad.r2.1.pca_loadings.ht'
+)
+
+
+@click.command()
+@click.option('--output', help='GCS output path', required=True)
+def query(output):  # pylint: disable=too-many-locals
+    """Query script entry point."""
+
+    hl.init(default_reference='GRCh38')
+
+    gnomad_loadings_path = f'{output}/gnomad_loadings_90k_liftover.ht'
+
+    # liftover and get variants
+    ht_gnomad_loadings = hl.read_table(GNOMAD_V2_LOADINGS)
+    ht_gnomad_loadings.count()
+    # 94176
```

Comment:
```suggestion
```

Reply:
Thanks for this. This is a hangover from sanity checks in R.